defender/utils.py

```diff
@@ -126,20 +127,42 @@ def get_username_blocked_cache_key(username):
     )
 
 
+def remove_prefix(string, prefix):
+
+    # backwards compatibility for str.removeprefix for python < 3.9
+    if sys.version_info < (3, 9):
+        if string.startswith(prefix):
+            return string[len(prefix) :]
+        return string
```

Comment:
Unit tests don't cover this line, which means we are missing a test for this condition.